docs/source/concepts/entry_points.rst

```diff
@@ -88,10 +88,10 @@ modification of a simulation defined in a model specification file.
    intended use case, so tread cautiously.
 
 By intention, the context exposes a very simple interface for managing the
-:ref:`simulation lifecycle <lifecycle_concept>`.  The combination of
```

Comment:
The fact that this was initially documented this way is the main reason I chose initially not to move the SimulationContext instantiation to cli.py. But here we are anyway